iotdb-core/datanode/src/main/java/org/apache/iotdb/db/tools/validate/TsFileOverlapValidationAndRepairTool.java

```diff
@@ -129,10 +129,9 @@ private static void moveSeqResourceToUnsequenceDir(TsFileResource resource) thro
     moveFile(
         new File(tsfile.getAbsolutePath() + TsFileResource.RESOURCE_SUFFIX),
         new File(targetFile.getAbsolutePath() + TsFileResource.RESOURCE_SUFFIX));
-    if (resource.modFileExists()) {
+    if (resource.anyModFileExists()) {
```

Comment:
If only shared mods file exists, it will also enter the following move method.

Reply:
Done